src/layers.cpp

```diff
@@ -3,15 +3,20 @@
 static uint8_t DefaultLayer;
 static uint32_t LayerState;
 
+// The maximum number of layers allowed. `LayerState`, which stores
+// the on/off status of the layers in a bitfield has only 32 bits, and
+// that should be enough for almost any layout.
+#define MAX_LAYERS sizeof(LayerState) * 8;
+
```

Comment:
I put this right below the declaration of `LayerState` to make things as obvious as possible.